### PR TITLE
User ratings and Store item Stats

### LINKED ISSUE #1334: [CWS CUJ] I understand how the impression of my item is determined
[Friction Log](https://docs.google.com/document/d/1S9Ik-ev8YdsO4gwAl_s7mRbtaYrApt7fUMZ-ReYi3eA/edit?usp=sharing)

### LINKED ISSUE #1376: [CWS CUJ] I can see my extension ratings and comments
[Friction Log](https://docs.google.com/document/d/1pBaVEms4KjWa_WT2t_7JrcjACI0F8ePqi-om-1TPf18/edit?usp=sharing)

### LINKED ISSUE #1426: [CWS CUJS] I can see the downloads and user stats for my store items
[Friction Log](https://docs.google.com/document/d/10yNioTAOmh657adFC9Vwty31bwK7cZGgdpYcHer3oyM/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #1426 
Fixes #1376 
Fixes #1334

cc: @CharyCh 

Changes proposed in this pull request:

Add the following sections to [Manage your CWS item](https://developer.chrome.com/docs/webstore/manage/)
- Improve ratings and reply to user reviews
- Track your store listing performance

⭐ Staged link

[Manage your CWS item](https://deploy-preview-1427--developer-chrome-com.netlify.app/docs/webstore/manage/)

### CODE REVIEW

File: site/en/docs/webstore/manage/index.md [hunk 1]
```diff
@@ -62,6 +63,98 @@ you can remedy the problem check the [Troubleshooting guide][troubleshooting].
...
+Users are more likely to improve their review if you respond and address their concerns promptly. To
+receive email notifications when users post reviews or update existing reviews, you can enable item
+reviews in the **Account settings** of the Developer Dashboard.
+
+
+   {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/8KGlMCKpiVVBRVqPAkXF.png", alt="Enable reviews notifications", width="600", height="212" %}
+
+The **Rating tab** in the developer dashboard provides an overview of your extension’s ratings,
+including the total number of reviews and ratings over time.
+e.
```
Comment (samthor) [inline site/en/docs/webstore/manage/index.md:95]: trailing "e."

**Review by samthor — APPROVED** [review]
Not a TW review but LGTM to me. cc @awfuchs

File: site/en/docs/webstore/manage/index.md [hunk 2]
```diff
@@ -62,6 +63,98 @@ you can remedy the problem check the [Troubleshooting guide][troubleshooting].
...
+
+You can monitor weekly user retention for different groups of users, categorized by country,
+language, operating system, and item version.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/jOUcBDD8sBcTFCXrGq1s.png", alt="weekly users
+statistics chart", width="800", height="342" %}
+
+{% Aside %}
+
+The Weekly users stats does not monitor active users.
```
Comment (awfuchs) [inline site/en/docs/webstore/manage/index.md:146]: Either "Weekly Users" so case agrees with the UI, or "weekly users" to use them as regular words.

Comment (awfuchs) [reply site/en/docs/webstore/manage/index.md:146]: This sounds as though active users are excluded from the count. Perhaps "only captures installations; it doesn't monitor whether users are active or not" -- if that is what was intended.

File: site/en/docs/webstore/manage/index.md (hunk 1 above)
Comment (awfuchs) [reply site/en/docs/webstore/manage/index.md:95]: Yeah, looks gnarly in the preview!

Comment (AmySteam) [reply site/en/docs/webstore/manage/index.md:95]: 🤣 Just to keep ya'll on your toes!

File: site/en/docs/webstore/manage/index.md (hunk 2 above)
Comment (AmySteam) [reply site/en/docs/webstore/manage/index.md:146]: Much better thanks!

**Pull request merged**